lib/const.rb

```diff
@@ -1,6 +1,6 @@
 # Defines common constants used in different parts of crew
 
-CREW_VERSION = '1.26.2'
+CREW_VERSION = '1.26.3'
```

Comment:
With so many changes, it might be justified to bump to `1.27.0`?

Reply:
done